packages/hurl/src/runner/result.rs

```diff
@@ -141,34 +143,45 @@ impl EntryResult {
         stdout: &mut Stdout,
         source_info: SourceInfo,
     ) -> Result<(), Error> {
-        let Some(call) = self.calls.last() else {
-            return Ok(());
-        };
-        // We check file access authorization for file output when a context dir has been given
-        if let Output::File(filename) = output {
-            if !context_dir.is_access_allowed(filename) {
-                let inner = RunnerError::UnauthorizedFileAccess {
-                    path: PathBuf::from(filename.clone()),
-                };
-                return Err(Error::new(source_info, inner, false));
+        if let Some(call) = self.calls.last() {
```

Comment:
Why do we change the `let-else` for an `if let`? If there is no "real" benenit, I prefer to have the minimal change.